docs/workforce-management-framework/topics/pro-using-demo-app.adoc

```diff
@@ -0,0 +1,110 @@
+= Using the demo application user interface
+
+The {WFM-RC-LongName} demo application consists of three components:
+
+- A mobile application for end users
+- A portal application for administrator users on a web browser
+- A cloud application for backend server functionality (No user interface)
+
+The demo application has demo users and demo data to show off the functionality of {WFM-RC-LongName}.
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as workorders. Workers can view their workorder via a mobile applicaion and administrators
+can managed workorders via the web portal.
+
+== Mobile application
+
+The mobile application is used by a end user to interact with workorders that have been assigned to them.
+login screen requires username and password. For a list of users see 
+link:https://github.com/feedhenry-raincatcher/raincatcher-core/blob/master/demo/server/src/modules/wfm-user/users.json[users.json],
+all users in the demo have a default password is `123`
+
+image::/shared/images/login.png[Login]
+
+Once logged in you will be presented with the workorders screen which will show the workorders assigned to the logged in user.
+In this example workorders are accident reports to be filled out at the seen of an accident.
```

Comment:
That;s not example. Say taht demo data has workorders that are accident reports.

Reply:
Example is referring to the screen shot. I should make it clearer .

```In this example in the screen shot below, workorders are accident reports to be filled out at the seen of an accident.```